File: python3/1011.py
class Esfera:
    def __init__(self, radius):
        self.radius = float(radius)

    def volume(self):
        return (4.0/3.0)*3.14159*(self.radius)**3

esfera = Esfera(input())
print("VOLUME =", format(esfera.volume(), '.3f'))

File: solutions/uri/1 - Iniciante/1011/1011.rb
class Esfera
	@radius
	def initialize(radius)
		@radius = radius.to_f
	end
	
	def volume
		(4.0/3.0)*3.14159*@radius**3
	end
end

esfera = Esfera.new(gets.chomp)
puts "VOLUME = " + '%.3f' % esfera.volume